addons/account/views/res_config_settings_views.xml

```diff
@@ -145,26 +145,11 @@
                                         <div class="row mt16">
                                             <label for="currency_id" class="col-lg-3 o_light_label"/>
                                             <field name="currency_id" options="{'no_create_edit': True, 'no_open': True}" context="{'active_test': False}"/>
+                                            <field name="group_multi_currency" invisible="1"/>
                                         </div>
                                     </div>
                                 </div>
                             </div>
-                            <div class="col-12 col-lg-6 o_setting_box" id="record_transactions">
-                                <div class="o_setting_left_pane">
-                                    <field name="group_multi_currency"/>
```

Comment:
Imo, this field should be removed and all other places where it's used should be removed as well because this "new" behavior should be applied everywhere, not only in accounting

Reply:
removed from settings of website_sale and point_of_sale